examples/ruby/grpc-demo.gemspec

```diff
@@ -3,7 +3,7 @@
 
 Gem::Specification.new do |s|
   s.name          = 'grpc-demo'
-  s.version       = '0.11.0'
```

Comment:
Since this is on the v1.0.x branch, we should probably update this version and the dependency to 1.0.0.